app/src/main/res/values-de/strings.xml

```diff
@@ -92,7 +83,7 @@
     <string name="licences_summary">Rechtliche Hinweise zu Lizenzen</string>
 
     <!-- Startup Wizard -->
-    <string name="token_not_enabled">Nicht angemeldet</string>
+    <string name="not_connected_to_tumonline">Nicht angemeldet</string>
```

Comment:
This is not a proper translation of "Not connected"

Reply:
I think “angemeldet” fits better than “verbunden”, but since we seem to use the latter throughout the app, I’ve changed this translation.